diff --git a/node_modules/expo-application/android/src/main/java/expo/modules/application/ApplicationModule.kt b/node_modules/expo-application/android/src/main/java/expo/modules/application/ApplicationModule.kt
index 8d63e9b..dfa3069 100644
--- a/node_modules/expo-application/android/src/main/java/expo/modules/application/ApplicationModule.kt
+++ b/node_modules/expo-application/android/src/main/java/expo/modules/application/ApplicationModule.kt
@@ -57,7 +57,7 @@ class ApplicationModule(private val mContext: Context) : ExportedModule(mContext
       Log.e(TAG, "Exception: ", e)
     }

-    constants["androidId"] = Settings.Secure.getString(mContext.contentResolver, Settings.Secure.ANDROID_ID)
+    // constants["androidId"] = Settings.Secure.getString(mContext.contentResolver, Settings.Secure.ANDROID_ID)

     return constants
   }
