diff --git a/node_modules/taro-css-to-react-native/dist/css-to-react-native/transforms/flex.js b/node_modules/taro-css-to-react-native/dist/css-to-react-native/transforms/flex.js
index 7d6223e..64c645b 100644
--- a/node_modules/taro-css-to-react-native/dist/css-to-react-native/transforms/flex.js
+++ b/node_modules/taro-css-to-react-native/dist/css-to-react-native/transforms/flex.js
@@ -14,6 +14,16 @@ var defaultFlexGrow = 1;
 var defaultFlexShrink = 1;
 var defaultFlexBasis = 0;
 var _default = function _default(tokenStream) {
+  /**
+   * fix: 将flex: 1;原样输出
+   */
+  if (tokenStream.nodes.length === 1 && tokenStream.nodes[0].value === '1') {
+    return {
+      $merge: {
+        flex: 1
+      }
+    };
+  }
   var flexGrow;
   var flexShrink;
   var flexBasis;
