diff --git a/node_modules/@tarojs/components/dist/components/taro-view-core.js b/node_modules/@tarojs/components/dist/components/taro-view-core.js
index 2f5ab06..3843441 100644
--- a/node_modules/@tarojs/components/dist/components/taro-view-core.js
+++ b/node_modules/@tarojs/components/dist/components/taro-view-core.js
@@ -1,7 +1,7 @@
 import { proxyCustomElement, HTMLElement, createEvent, h, Host } from '@stencil/core/internal/client';
 import { c as classnames } from './index2.js';
 
-const indexCss = "body,html{-webkit-user-select:none;-moz-user-select:none;-ms-user-select:none;user-select:none;-webkit-tap-highlight-color:rgba(0, 0, 0, 0)}taro-view-core{display:block}";
+const indexCss = "body,html{-webkit-user-select:none;-moz-user-select:none;-ms-user-select:none;user-select:none;-webkit-tap-highlight-color:rgba(0, 0, 0, 0)}";
 
 const View = /*@__PURE__*/ proxyCustomElement(class extends HTMLElement {
   constructor() {
