diff --git a/node_modules/@tarojs/components-advanced/dist/components/virtual-list/preset.js b/node_modules/@tarojs/components-advanced/dist/components/virtual-list/preset.js
index 3b64150..7fd6753 100644
--- a/node_modules/@tarojs/components-advanced/dist/components/virtual-list/preset.js
+++ b/node_modules/@tarojs/components-advanced/dist/components/virtual-list/preset.js
@@ -140,7 +140,7 @@ class Preset {
     boundaryDetection() {
         if ([typeof this.props.onScrollToUpper, typeof this.props.onScrollToLower].every(e => e !== 'function'))
             return;
-        createSelectorQuery().select(`#${this.id}`).node().exec(() => {
+        createSelectorQuery().select(`#${this.id}`).boundingClientRect().exec(() => {
             const upperObserver = this.boundaryDetectionHelper({
                 event: typeof this.props.onScrollToUpper === 'function' ? () => {
                     if (this.field.diffOffset >= 0)
diff --git a/node_modules/@tarojs/components-advanced/dist/components/virtual-list/react/list.js b/node_modules/@tarojs/components-advanced/dist/components/virtual-list/react/list.js
index a4c0d04..6462191 100644
--- a/node_modules/@tarojs/components-advanced/dist/components/virtual-list/react/list.js
+++ b/node_modules/@tarojs/components-advanced/dist/components/virtual-list/react/list.js
@@ -260,7 +260,7 @@ class List extends React.PureComponent {
             }
         }
         this._callPropsCallbacks();
-        this.preset.boundaryDetection();
+        // this.preset.boundaryDetection();
     }
     componentDidUpdate(prevProps, prevState) {
         const { scrollOffset, scrollUpdateWasRequested } = this.state;
@@ -378,8 +378,7 @@ class List extends React.PureComponent {
         const _a = omit(this.props, [
             'item', 'itemCount', 'itemData', 'itemKey', 'useIsScrolling',
             'innerElementType', 'innerTagName', 'itemElementType', 'itemTagName',
-            'outerElementType', 'outerTagName', 'onScrollToLower', 'onScrollToUpper',
-            'upperThreshold', 'lowerThreshold',
+            'outerElementType', 'outerTagName',
             'position', 'innerRef',
         ]), { className, direction, height, layout, style, width, enhanced = false, outerWrapper, renderTop, renderBottom } = _a, rest = __rest(_a, ["className", "direction", "height", "layout", "style", "width", "enhanced", "outerWrapper", "renderTop", "renderBottom"]);
         const { scrollOffset, scrollUpdateWasRequested } = this.state;
diff --git a/node_modules/@tarojs/components-advanced/dist/components/virtual-waterfall/preset.js b/node_modules/@tarojs/components-advanced/dist/components/virtual-waterfall/preset.js
index 3d27413..968b2d1 100644
--- a/node_modules/@tarojs/components-advanced/dist/components/virtual-waterfall/preset.js
+++ b/node_modules/@tarojs/components-advanced/dist/components/virtual-waterfall/preset.js
@@ -119,7 +119,7 @@ class Preset {
     boundaryDetection() {
         if ([typeof this.props.onScrollToUpper, typeof this.props.onScrollToLower].every(e => e !== 'function'))
             return;
-        createSelectorQuery().select(`#${this.id}`).node().exec(() => {
+          createSelectorQuery().select(`#${this.id}`).boundingClientRect().exec(() => {
             const upperObserver = this.boundaryDetectionHelper({
                 event: typeof this.props.onScrollToUpper === 'function' ? () => {
                     if (this.field.diffOffset >= 0)
diff --git a/node_modules/@tarojs/components-advanced/dist/components/virtual-waterfall/react/waterfall.js b/node_modules/@tarojs/components-advanced/dist/components/virtual-waterfall/react/waterfall.js
index 40fff15..2e65c27 100644
--- a/node_modules/@tarojs/components-advanced/dist/components/virtual-waterfall/react/waterfall.js
+++ b/node_modules/@tarojs/components-advanced/dist/components/virtual-waterfall/react/waterfall.js
@@ -194,7 +194,7 @@ class Waterfall extends React.PureComponent {
             this._outerRef.scrollTop = initialScrollOffset;
         }
         this._callPropsCallbacks();
-        this.preset.boundaryDetection();
+        // this.preset.boundaryDetection();
     }
     componentDidUpdate(prevProps, prevState) {
         const { scrollOffset, scrollUpdateWasRequested } = this.state;
@@ -285,8 +285,7 @@ class Waterfall extends React.PureComponent {
         const _a = omit(this.props, [
             'item', 'itemCount', 'itemData', 'itemKey', 'useIsScrolling',
             'innerElementType', 'innerTagName', 'itemElementType', 'itemTagName',
-            'outerElementType', 'outerTagName', 'onScrollToLower', 'onScrollToUpper',
-            'upperThreshold', 'lowerThreshold',
+            'outerElementType', 'outerTagName',
             'position',
         ]), { className, style, height, width, enhanced, outerWrapper, renderTop, renderBottom } = _a, rest = __rest(_a, ["className", "style", "height", "width", "enhanced", "outerWrapper", "renderTop", "renderBottom"]);
         const { isScrolling, scrollOffset, scrollUpdateWasRequested } = this.state;
@@ -303,8 +302,8 @@ class Waterfall extends React.PureComponent {
         return React.createElement(outerWrapper || this.preset.outerElement, outerProps, React.createElement(this.preset.innerElement, {
             key: `${this.preset.id}-wrapper`,
             id: `${this.preset.id}-wrapper`,
-            className: classNames(className, 'virtual-waterfall-wrapper'),
-            style: Object.assign({ display: 'flex', justifyContent: 'space-evenly', pointerEvents: isScrolling ? 'none' : 'auto', position: 'relative', height: estimatedHeight, width: '100%' }, style),
+            className: 'virtual-waterfall-wrapper',
+            style: { display: 'flex', justifyContent: 'space-evenly', pointerEvents: isScrolling ? 'none' : 'auto', position: 'relative', height: estimatedHeight, width: '100%' },
         }, columnNodes));
     }
 }
diff --git a/node_modules/@tarojs/components-advanced/dist/components/virtual-waterfall/react/wrapper.js b/node_modules/@tarojs/components-advanced/dist/components/virtual-waterfall/react/wrapper.js
index 5b43a11..da61e5c 100644
--- a/node_modules/@tarojs/components-advanced/dist/components/virtual-waterfall/react/wrapper.js
+++ b/node_modules/@tarojs/components-advanced/dist/components/virtual-waterfall/react/wrapper.js
@@ -34,15 +34,15 @@ const outerWrapper = React.forwardRef(function OuterWrapper(props, ref) {
         getRenderExpandNodes({
             direction: 'top',
             id,
-            innerElement: innerElementType,
-            renderExpand: renderTop,
+            innerElement: innerElementType
         }),
+        renderTop,
         children,
+        renderBottom,
         getRenderExpandNodes({
             direction: 'bottom',
             id,
-            innerElement: innerElementType,
-            renderExpand: renderBottom,
+            innerElement: innerElementType
         }),
     ]);
 });
