diff --git a/node_modules/@tarojs/components/dist/components/taro-view-core.js b/node_modules/@tarojs/components/dist/components/taro-view-core.js
index 3e511ff..4506300 100644
--- a/node_modules/@tarojs/components/dist/components/taro-view-core.js
+++ b/node_modules/@tarojs/components/dist/components/taro-view-core.js
@@ -2,7 +2,7 @@ import { proxyCustomElement, HTMLElement, createEvent, h, Host } from '@stencil/
 import { c as classnames } from './index2.js';
 import { h as handleStencilNodes } from './helper.js';

-const indexCss = "taro-view-core{display:block}";
+const indexCss = "";

 const View = /*@__PURE__*/ proxyCustomElement(class extends HTMLElement {
   constructor() {
