diff --git a/node_modules/@tarojs/plugin-platform-tt/dist/index.js b/node_modules/@tarojs/plugin-platform-tt/dist/index.js
index f9a6990..430be1b 100644
--- a/node_modules/@tarojs/plugin-platform-tt/dist/index.js
+++ b/node_modules/@tarojs/plugin-platform-tt/dist/index.js
@@ -39,6 +39,9 @@ const components = {
     Icon: {
         size: '24'
     },
+    View: {
+        bindClick: 'bindTap'
+    },
     Button: {
         bindGetPhoneNumber: _empty,
         'data-channel': _empty
